aea/crypto/base.py

```diff
@@ -443,6 +443,44 @@ def update_with_gas_estimate(self, transaction: JSONLike) -> JSONLike:
         :return: the updated transaction
         """
 
+    @abstractmethod
+    def contract_method_call(
+        self, contract_interface: Dict[str, str], method_name: str, **kwargs: Any,
+    ) -> JSONLike:
+        """Call method"""
```

Comment:
We also need to implement in the LedgerAPI base to access it from the contract base, right?

Reply:
Yes you need to define the abstract methods you want to be implemented in the plugins. For the other plugins you can raise NotImplementedError for now